playwright/flaky-reporter.ts

```diff
@@ -49,18 +104,12 @@ class FlakyReporter implements Reporter {
         const { GITHUB_TOKEN, GITHUB_API_URL, GITHUB_SERVER_URL, GITHUB_REPOSITORY, GITHUB_RUN_ID } = process.env;
         if (!GITHUB_TOKEN) return;
 
-        const body = `${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}`;
-
-        const headers = { Authorization: `Bearer ${GITHUB_TOKEN}` };
-        // Fetch all existing issues with the flaky-test label.
-        const issuesRequest = await fetch(
-            `${GITHUB_API_URL}/repos/${REPO}/issues?labels=${LABEL}&state=all&per_page=100&sort=created`,
-            { headers },
-        );
-        const issues = await issuesRequest.json();
+        const issues = await this.getAllIssues();
         for (const flake of this.flakes) {
             const title = ISSUE_TITLE_PREFIX + "`" + flake + "`";
             const existingIssue = issues.find((issue) => issue.title === title);
+            const headers = { Authorization: `Bearer ${GITHUB_TOKEN}` };
+            const body = `${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}`;
```

Comment:
why did this move into the for loop? tiny de-optimisation

Reply:
didn't think much of it, just moved the variables closer to where they're used. I'm sure the compiler will move stuff out for us anyway :crossed_fingers: